### PR TITLE
Add playerctl v2 backends

### PR DESCRIPTION
**Pull request by Kasper24**

### CODE REVIEW

**Review by HumblePresent — APPROVED** [review]

**Review by HumblePresent — APPROVED** [review]

**Comment by Kasper24** [comment]
@JavaCafe01 Can you review my the doc changes?

File: docs/signals/pctl.md [hunk 1]
```diff
@@ -17,27 +17,65 @@ This module relies on `playerctl` and `curl`. If you have this module disabled,
 
 ### Usage
 
-To enable: `bling.signal.playerctl.enable()`
+To enable: `playerctl = bling.signal.playerctl.lib/cli()`
```
Comment (JavaCafe01) [inline docs/signals/pctl.md:20]:
```suggestion
To connect to bling's `playerctl` module: `playerctl = bling.signal.playerctl.lib()` or `playerctl = bling.signal.playerctl.cli()`

Throughout the documentation, we will refer to both the lib and the cli as `bling.signal.playerctl.lib/cli()`.
```

**Review by JavaCafe01 — CHANGES_REQUESTED** [review]
Just some minor wording fixes in the docs as addressed in my comment. If you're ok with it, go ahead and merge this PR.

**Review by JavaCafe01 — APPROVED** [review]

**Comment by JavaCafe01** [comment]
@Nooo37 Good for merge?

**Comment by nawuko** [comment]
Tested this PR since 5 days (lib variant), had no problems so far, only thing i noticed is that with the new signals, the metadata is already escaped & -> &amp;
This is nice so users don't have todo that manually, sadly if you don't use a complete custom widget for notifications (so any naughty pre-defined widget for title/message etc) it gets escaped again, meaning &amp; shows up in the status notification [&amp;amp;], would be awesome to have an option to get only "raw"/unescaped metadata via signals for use cases like that.

**Review by Nooo37 — APPROVED** [review]
Works fine for my limited usecase + no obvious compatibility issues :+1: 

@Kasper24  What do you think about nawukos comment?

**Comment by Kasper24** [comment]
@nawuko I actually ran into the same issue in my notification center widget, so I had to unescape it before escaping again. I'm not sure what we should do by default?
- Don't escape it
- Add an option to enable escaping (should it be on or off by default?)
- Add another signal i.e metadata_raw
- Leave as is?
@Nooo37 @JavaCafe01

**Comment by JavaCafe01** [comment]
Can't seem to get it working. I first did get the notification to not use the old signals anymore, but after changing everything to the new methods and signals my widgets never updated or showed any song info.




shot_1216154102


https://user-images.githubusercontent.com/33443763/146467501-c392b383-ccbb-4b8c-9ad9-bf0698476ab8.mp4

**Comment by JavaCafe01** [comment]
Gonna merge as it works for everyone but me :(

**Pull request merged**